docs/source/api_reference/layers.rst

```diff
@@ -207,6 +207,9 @@ Torch Layers
 .. autoclass:: deepchem.models.torch_models.layers.DTNNEmbedding
   :members:
 
+.. autoclass:: deepchem.models.torch_models.layers.WeaveLayer
```

Comment:
Please make sure to add to https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/torch_layers.csv as well

Reply:
Sir I have added it now.